addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -579,7 +581,12 @@ const UserValueWidget = Widget.extend({
      * @param {boolean} show
      */
     toggleVisibility: function (show) {
+        const wasInvisible = this.el.classList.contains('d-none');
         this.el.classList.toggle('d-none', !show);
+        const requestFocus = this.el.dataset.requestFocus;
+        if (show && wasInvisible && requestFocus && requestFocus === "true") {
```

Comment:
Falsy `requestFocus` will also be rejected by `requestFocus === "true"`, I think that precondition can be removed.